frontend_app_html-document/src/container/HtmlDocument.jsx

```diff
@@ -116,6 +116,7 @@ class HtmlDocument extends React.Component {
       rawContentBeforeEdit: prev.content.raw_content,
       timeline: addRevisionFromTLM(data, prev.timeline, state.loggedUser.lang)
     }))
+    GLOBAL_dispatchEvent({ type: CUSTOM_EVENT.REFRESH_CONTENT_LIST, data: {} })
```

Comment:
these will end up obsolete since the components that rely on this custom event will now rely on the TLM.
So, I'm ok to keep it until then but it needs a comment with a TODO or FIXME

Reply:
Changed